bot/constants.py

```diff
@@ -0,0 +1,34 @@
+from os import environ
+from typing import NamedTuple
+
+
+class Channels(NamedTuple):
```

Comment:
Note, doesn't have to be for this PR, but I think we can nix most of the channel overrides. I think the ones worth keeping are:
- bot commands
- lancebot playground

... that's it. The rest can be added in with the feature sets.

Reply:
Gone with [b3458b5](https://github.com/python-discord/sir-robin/pull/10/commits/b3458b5658f686acebc444bc128151a558df7810)